spec/models/page_spec.rb

```diff
@@ -165,18 +165,20 @@
     end
   end
 
-  describe 'Thumbnail' do
-    let(:slug) { 'reducing-paperwork' }
+  describe '#thumbnail' do
+    context 'when help page with image' do
```

Comment:
`'when page has image'` or `'with linked asset'`
vs.
`'when page has no image'` or `'without linked asset'`

The first makes for grammatically correct spec docs but the latter is even better.
1. context
2. with or without
3. what specifically